packages/xstate-react/src/useInterpret.ts

```diff
@@ -32,40 +29,21 @@ export function useIdleInterpreter(
   ) {
     const [initialMachine] = useState(machine);
 
-    if (getMachine !== initialMachine) {
+    if (getMachine.config !== initialMachine.config) {
```

Comment:
Similarly, `.config` is super machine-specific thing. If we want to make `useInterpret` to accept just any actor then it should be possible to check smth like this on just any actor type.

Reply:
Good call, I'll think about how we can generalize this.

Reply:
Added to 4659f4511539b6c5c6ed4b585796aed0eef0f8e8 - let me know what you think. Might need help on this